responses.py

```diff
@@ -253,6 +260,18 @@ def match(request_body):
     return match
 
 
+def json_keys_matcher(keys):
```

Comment:
I wonder if all the matchers should be moved into another module. Perhaps `responses.match`? That could be done independently of these changes too though.